owncloudApp/src/main/java/com/owncloud/android/ui/errorhandling/ErrorMessageAdapter.kt

```diff
@@ -129,11 +129,11 @@ class ErrorMessageAdapter {
                     if (operation is CreateFolderOperation) f.forbidden(R.string.forbidden_permissions_create)
                     if (operation is MoveFileOperation) f.forbidden(R.string.forbidden_permissions_move)
                     if (operation is CopyFileOperation) f.forbidden(R.string.forbidden_permissions_copy) else f.format(
-                        R.string.filename_forbidden_charaters_from_server
```

Comment:
this probably breaks localizations. I generally don't touch localizations as projects typically have special rules for updates.

Reply:
Yes, it will unless we rename it on every single strings file.

So, we have two options:
1. Rename it on every single strings file on this PR
2. Revert this change and rename it on another PR

Choose the one you are more comfortable with.

Reply:
Changes applied, renamed on every strings file 👍